lesson_three/two.py

```diff
@@ -0,0 +1,20 @@
+"""
+3. В массиве случайных целых чисел поменять местами минимальный и максимальный элементы.
+"""
+
+from random import randint
+
+_list_ = [randint(1, 99) for _ in range(10)]
```

Comment:
у "_" перед названием есть определенный смысл, здесь оно не нужно
```suggestion
list_ = [randint(1, 99) for _ in range(10)]
```